go/src/koding/klientctl/endpoint/stack/stack.go

```diff
@@ -35,22 +36,21 @@ func (opts *CreateOptions) Valid() error {
 	return nil
 }
 
-// DefaultClient
-var DefaultClient Client
+var DefaultClient = &Client{}
 
-// Client
 type Client struct {
 	Kloud      *kloud.Client
 	Credential *credential.Client
 }
 
-// Create
 func (c *Client) Create(opts *CreateOptions) (*stack.ImportResponse, error) {
 	if err := opts.Valid(); err != nil {
 		return nil, err
 	}
 
+	log.Println("BEFORE RE-ENCODE")
 	data, err := c.jsonReencode(opts.Template)
+	log.Println("AFTER RE-ENCODE")
```

Comment:
I might be wrong but this looks like debug prints. Are you going to leave them here?